### PR TITLE
Update labels and URLs for outdated-icons

### PR DESCRIPTION
**Pull request by LitoMore**
The label in URLs should be updated since we renamed it.

### CODE REVIEW

File: DISCLAIMER.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Should a brand wish for their icon or data to be updated, please [submit an issu
 Should a brand wish for their icon to be removed from the package, contact `removals at simpleicons dot org` explaining your affiliation with the company, and reasons for removal. Alternatively, it is also possible to [submit an issue][removal-issues] on [the Simple Icons GitHub repository] with the same information. We generally remove icons that no longer [meet our criteria] twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.
 
 [meet our criteria]: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon
-[icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues/new?assignees=&labels=icon+outdated&template=icon_update.yml&title=Update%3A+
+[icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues/new?labels=update+icon%2Fdata&template=icon_update.yml&title=Update%3A+
```
Comment (mondeja) [inline DISCLAIMER.md:50]:
```suggestion
[icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues/new?labels=update%20icon%2Fdata&template=icon_update.yml&title=Update%3A+
```

File: .github/ISSUE_TEMPLATE/icon_update.yml [hunk 2]
```diff
@@ -8,11 +8,11 @@ body:
     attributes:
       value: |
         Before opening a new issue, make sure it isn't covered by an existing issue.
-        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
+        Please search for [issues with the `update-icon/data` label][update-icon/data]
```
Comment (mondeja) [inline .github/ISSUE_TEMPLATE/icon_update.yml:11]:
```suggestion
        Please search for [issues with the `update icon/data` label][update-icon/data]
```

File: .github/ISSUE_TEMPLATE/icon_update.yml [hunk 3]
```diff
@@ -8,11 +8,11 @@ body:
...
         (including closed issues) before you continue. If you find one for the brand
         you're reporting then leave a comment on it or add a reaction.
 
-        [icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22
+        [update-icon/data]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22update+icon%2Fdata%22
```
Comment (mondeja) [inline .github/ISSUE_TEMPLATE/icon_update.yml:15]: This URL works but seems more correct to me using ` ` (space, %20):

```suggestion
        [update-icon/data]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22update%20icon%2Fdata%22
```

**Review by mondeja — CHANGES_REQUESTED** [review]
Additionally to the following comments, the field `labels` of the template *.github/ISSUE_TEMPLATE/icon_update.yml* must be updated.

File: .github/ISSUE_TEMPLATE/icon_update.yml (hunk 3 above)
Comment (LitoMore) [reply .github/ISSUE_TEMPLATE/icon_update.yml:15]: The `+` is more correct. Even GitHub is using the `+` for spaces. You can go to our issues page and select a label for the filter. The URL will be:

```
https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aassessing
```

Comment (mondeja) [reply .github/ISSUE_TEMPLATE/icon_update.yml:15]: Looks like a crazy decission of GH to me but not a big deal.

**Comment by LitoMore** [comment]
I still have concerns about those label naming changes.

I feel that our labels are not as accurate as before. We can no longer find all issues related to icon updates or icon removal through a certain label. The semantics are not as easy to understand as before. These may not be friendly to our first-time contributors and issue authors.

If you want to adopt these changes, you can review this one instead: 

- #10804

File: .github/ISSUE_TEMPLATE/icon_update.yml (hunk 2 above)
Comment (LitoMore) [reply .github/ISSUE_TEMPLATE/icon_update.yml:11]: Yes, let's drop the PR in favor of @adamrusted's #10804.

**Pull request closed without merging**